# Computational-biology-project-

Research Goals:

Investigated how interconnected biological features (transcriptome and/or metabolome) change across different locations (D01-D12), concentration levels (Control, 1x, 10x), and their interactions.

Integrative Analysis:

Performed integrative analysis on metabolomics and transcriptomics data using advanced machine learning techniques.
Applied PLS-DA, RGCCA, and iterative random forest to extract meaningful biological insights.
Annotated results to interpret relationships between metabolomic and transcriptomic datasets.

Omics Feature Annotation:

Mapped Daphnia magna genes to other model species for more convenient annotation.
Conducted ortholog mapping using the latest version of OrthoDB to Daphnia pulex
Annotated peaks in the metabolome spectrum using BEAMSpy for both positive and negative modes. This annotation utilized KEGG and HMDB
Provided key insights into complex biological processes through data-driven approaches. Additionally, MetaboAnalyst was employed for enrichment analysis of the metabolomics data.
